app/api/custom/orders.py

```diff
@@ -82,8 +88,11 @@ def resend_emails():
 
 @order_blueprint.route('/calculate-amount', methods=['POST'])
 @jwt_required
-def calculate_amount():
-    data = request.get_json()
+def calculate_amount(*args):
+    if not args:
+        data = request.get_json()
+    else:
+        data = json.loads(args[0])
```

Comment:
As mentioned previously, this should not be done

Reply:
@iamareebjamal https://github.com/fossasia/open-event-server/pull/6635#issuecomment-561835218

Reply:
No benefit of merging an intermediate state PR

Reply:
@iamareebjamal I will start working on that PR as soon as this gets merged. I wanted to keep them separate as this PR already contains a big change. And the second change is more of a refactor.

Reply:
Yes, that's why it can be taken in this PR only. It's not a huge change compared to what has been done already. It makes no sense to break something first and then fix later in a separate PR